.rubocop.yml

```diff
@@ -81,6 +81,16 @@ Style/DotPosition:
 Style/DoubleNegation:
   Enabled: false
 
+Style/SymbolArray:
+  Enabled: false
+
+Style/IndentHeredoc:
```

Comment:
This explanation is great. Can you add it as a comment above the  config item so we can easily see why we did it?